internal/route/tree_test.go

```diff
@@ -381,7 +381,7 @@ func TestTree_Match(t *testing.T) {
 		},
 		{
 			path:       "/webapi//",
-			wantOK:     true,
+			wantOK:     false,
```

Comment:
We need to move this test case further down to the "No match" group after line 502.